manual/cops_style.md

```diff
@@ -164,6 +164,60 @@ EnforcedStyle | `always` | `always`, `conditionals`
 
 * [https://rubystyle.guide#no-and-or-or](https://rubystyle.guide#no-and-or-or)
 
+## Style/ArgumentsForwarding
+
+Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
+--- | --- | --- | --- | ---
+Enabled | Yes | Yes  | 0.80 | -
+
+In Ruby 2.7, arguments forwarding has been added.
+
+This cop identifies places where `do_something(*args, &block)`
+can be replaced by `do_something(...)`.
+
+### Examples
+
+```ruby
+# bad
+def foo(*args, &block)
+  bar(*args, &block)
+end
+
+# bad
+def foo(*args, **kwargs, &block)
+  bar(*args, **kwargs, &block)
+end
+
+# good
+def foo(...)
+  bar(...)
+end
+```
+#### Strict: true
+
+```ruby
+# bad
+# The following code can replace the arguments with `...`,
+# but it will change the behavior. Because `...` forwards block also.
+def foo(*args)
+  bar(*args)
+end
```

Comment:
I'm wondering if the "Strict" option is confusing. 
I meant when `Strict: true`, the cop adds offenses if it does not change behavior. When `Strict : false`, the cop adds an offense even if it may changes the behavior.
But it looks opposite to the implementation.


```ruby
# I meant the cop does not add an offense to the following code, if `Strict: true`.
# But the implementation and documentation say the opposite.
def foo(*args)
  bar(*args)
end
```

So, I guess we can invert the meaning, change the option name, or do not change anything.

Reply:
I updated the option name to `AllowOnlyRestArgument`.